### PR TITLE
[sqlcipher] update to 4.5.2

### PR DESCRIPTION
**Pull request by MSP-Greg**
**Describe the pull request**

- #### What does your PR fix?
  Fixes / Updates sqlcipher to 4.5.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  no change

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/sqlcipher/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sqlcipher",
-  "version": "4.5.1",
+  "version": "4.5.2",
   "description": "SQLCipher extends the SQLite database library to add security enhancements that make it more suitable for encrypted local data storage.",
   "homepage": "https://www.zetetic.net/sqlcipher",
```
Comment (Adela0814) [inline ports/sqlcipher/vcpkg.json:5]: Please add a license field.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/sqlcipher/vcpkg.json (hunk 1 above)
Comment (MSP-Greg) [reply ports/sqlcipher/vcpkg.json:5]: Done.  Proprietary license, so null...

File: ports/sqlcipher/portfile.cmake [hunk 2]
```diff
@@ -55,16 +55,15 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
...
     OPTIONS_DEBUG
         -DSQLITE3_SKIP_TOOLS=ON
 )
 
-vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/${PORT} TARGET_PATH share/${PORT})
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME ${PORT} CONFIG_PATH share/${PORT})
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
```
[the remark this answers is not in the document]
Comment (Adela0814) [reply ports/sqlcipher/portfile.cmake:68]: Can you please add quotes around each path that starts with `${CURRENT_PACKAGES_DIR}` or `${SOURCE_PATH}`?

**Review by Adela0814 — DISMISSED** [review]

File: ports/sqlcipher/portfile.cmake (hunk 2 above)
[the remark this answers is not in the document]
Comment (MSP-Greg) [reply ports/sqlcipher/portfile.cmake]: Done.

**Pull request merged**